integration-tests/integration-tests-quarkus-gradle/src/main/resources/gradle.properties

```
@@ -0,0 +1,7 @@
+# Gradle properties
+quarkusPluginId=io.quarkus
+quarkusPluginVersion=2.1.1.Final
```

Is it possible to use `${version.io.quarkus}` also here?
```suggestion
quarkusPluginVersion=${version.io.quarkus}
```

---

right! :)